newrelic/resource_newrelic_synthetics_monitor.go

```diff
@@ -95,158 +94,411 @@ func resourceNewRelicSyntheticsMonitor() *schema.Resource {
 				Optional:    true,
 				Description: "Fail the monitor check if redirected.",
 			},
+			"runtime_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "",
+			},
+			"runtime_type_version": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "",
+			},
+			"script_language": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "",
+			},
+			"tags": {
```

Comment:
When creating the migration guide we will need to make it clear that this might interfere with existing use of `newrelic_entity` and `newrelic_entity_tags`. As that's the previous way customers would set tags on new relic synthetic checks.